doc/datastorage_setup.md

```diff
@@ -4,50 +4,72 @@
 This section provides how to download and install zeromq
 
 #### 1. Install the dependency packages
-- sudo apt-get install libtool pkg-config build-essential autoconf   automake
-
-- sudo apt-get install libzmq3-dev
+```
+sudo apt-get install libtool pkg-config build-essential autoconf   automake
+sudo apt-get install libzmq3-dev
+```
 
 #### 2. Install libsodium (Zero MQ has dependcy on this package)
-1. Download the tar from https://libsodium.gitbook.io/doc/installation (Version: 1.0.18)
-2. tar -xvf tar -xvf libsodium_1.0.18.orig.tar.gz 
-3. cd libsodium-1.0.18
-4. ./autogen.sh 
+
```

Comment:
```suggestion
```
@suresh-lc If there is no intention to add a blank-line, then what about removing it like this?